operator.yaml

```diff
@@ -2865,9 +2873,9 @@ apiVersion: v1
 data:
   DEFAULT_BUILDER_RESOURCE_NAME: Dockerfile
   DEFAULT_WORKFLOW_EXTENSION: .sw.json
-  Dockerfile: "FROM quay.io/kiegroup/kogito-swf-builder-nightly:latest AS builder\n
-    \ \n  # Copy from build context to skeleton resources project\nCOPY * ./resources/\n\nRUN
-    /home/kogito/launch/build-app.sh ./resources\n  \n  #=============================\n
+  Dockerfile: "FROM quay.io/kiegroup/kogito-swf-builder:{{.BuilderImageVersion}} AS
```

Comment:
Because this dockerfile is not used by docker with  "docker build -t example-image:tag --build-arg VAR=value1" but is used by kaniko and in the next iteration by buildah